charger/abl.go

```diff
@@ -31,20 +31,16 @@ import (
 type ABLeMH struct {
 	log  *util.Logger
 	conn *modbus.Connection
+	curr uint16
 }
 
 const (
 	ablRegFirmware   = 0x01
 	ablRegStatus     = 0x04
```

Comment:
Kann eigentlich auch weg. Die Daten sind auch in 2E enthalten.

Reply:
bei "enable" muss ich den letzten Strom schreiben. der steht nicht in 2E

Reply:
Doch, schau mal in der Doku

Reply:
Und was soll ich setzen wenn da `3E8` drin steht???

Reply:
Das gleiche was du gesetzt hättest was in Reg 4 drin steht ;)
Wohl am besten lokal "merken" oder auf Minimum setzten.

Reply:
Häh? Ich merke es mir doch in curr???

Reply:
Die Anmerkung bezog sich auf die Verwendung von `ablRegStatus     = 0x04`